gameArchitecture/dataManager.py

```diff
@@ -26,7 +26,7 @@ def level_dummy_data() -> level_data:
         word_data = ["Coconuts", "Make", "Me", "Giggle"]
         for x in range(10):
             route.append((9 * 50, (9 - x) * 50))
-            towers.append((350, x * 50))
+            towers.append((350, 150))
```

Comment:
Not super pertinent, but when we start laying out the levels, we'll need to assign this based on data.

Reply:
Alright, sounds good! I just did that for testing purposes.